dojo/filters.py

```diff
@@ -27,10 +28,39 @@
 EARLIEST_FINDING = None
 
 
+def filter_by_cves(queryset, name, value):
+    values = value.split(',')
+    return queryset.filter(cve__in=values)
+
+
+def filter_by_numerical_severities(queryset, name, value):
+    values = value.split(',')
+    return queryset.filter(numerical_severity__in=values)
+
+
+def filter_by_severities(queryset, name, value):
+    values = value.split(',')
+    return queryset.filter(severity__in=values)
+
+
+def filter_by_test__engagement__product(queryset, name, value):
+    values = value.split(',')
+    return queryset.filter(test__engagement__product__in=values)
+
+
+def filter_by_test__test__type(queryset, name, value):
+    values = value.split(',')
+    return queryset.filter(test__test__type__in=values)
+
```

Comment:
Could this be refactored to have less code duplication? The `test__test__type__in=values` could maybe be turned into a Q  object, or a function that returns a Q object?

Reply:
`filter_by_test__engagement__product` and `filter_by_test__test__type` were left-overs thanks for pointing that out, I removed them.
I also made the code a bit DRYer using a Q filter (the doc is a pain seriously it took me 3 hours at least :sweat_smile: )

Reply:
Yeah, that `Q(**{..}` is always fun, but looks nice and tidy now!